core/mondoo-macos-security.mql.yaml

```diff
@@ -978,3 +981,79 @@ queries:
         Some legacy POSIX software might expect an available root account.
     query: |
       command('dscl . -read /Users/root AuthenticationAuthority') { stderr == /No such key: AuthenticationAuthority/ }
+  - uid: mondoo-macos-security-software-updates-automatic-check-enabled
+    title: Ensure automatic checking of software updates enabled
+    docs:
+      desc: |
+        This check determines where automatic checking for software updates is enabled
+      remediation: |-
+        Perform the following to enable automatic checking for updates:
+
+        _Graphical Method:_
+
+        1.  Open **System Preferences**.
+        2.  Select **Software Update**.
+        3.  Select **Advanced**.
+        4.  Select **Check for updates**.
+
+        _Terminal Method:_
+
+        Run the following command to enable auto update:
+
+        ```bash
+        sudo /usr/bin/defaults write /Library/Preferences/com.apple.SoftwareUpdate AutomaticCheckEnabled -bool true
+        ```
+    query: |
+      parse.plist('/Library/Preferences/com.apple.SoftwareUpdate.plist').
```

Comment:
the check needs to be "!= false", since the default is true and it is not set on my machine, although it is active. same applies for all other rules

Reply:
I updated the policies to cover the default setting where the values are not set